packages/camera/test/new/camera_test.dart

```diff
@@ -0,0 +1,139 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
```

Comment:
The `lib/new` currently has no functionality and will most likely be removed in: https://github.com/flutter/plugins/pull/3229. So, you can just delete this file. And you don't need to add them back to `lib/camera_test.dart`.